File: 6. Python/EX47.py
# Função para popular um vetor com valores do usuário
def popVetor(tam, txt, vet):
    print(f"\n{txt}")

    for i in range(tam):
        vet[i] = int(input(f"# Valor {[i+1]}: "))  # Coleta valor do vetor

    return vet

# Solicita a quantidade de valores para os vetores
qtd = int(input("# Quantos valores vai ter cada vetor: "))

# Cria vetores A, B e C
vetA = [0 for i in range(qtd)]
vetB = [0 for i in range(qtd)]
vetC = [0 for i in range(qtd)]

# Popula os vetores A e B
popVetor(qtd, "# Digite os valores do vetor A: ", vetA)
popVetor(qtd, "# Digite os valores do vetor B: ", vetB)

# Exibe o vetor resultante C
print("\n# VETOR RESULTANTE: ")
for i in range(qtd):
    vetC[i] = vetA[i] + vetB[i]  # Soma os vetores A e B
    print(f"-> {vetC[i]}")       # Exibe valor do vetor C

File: 4. Java/EX47.java
package Apostila;

import java.util.Scanner;

public class EX47
{
    public static void main(String[] args)
    {
        // Quantidade de elementos
        int qtd;
        
        // Lê a quantidade de valores para os vetores
        try (Scanner ent = new Scanner(System.in))
        {
            System.out.print("# Quantos valores vai ter cada vetor: ");
            qtd = ent.nextInt();

            double[] vetA = new double[qtd], vetB = new double[qtd], vetC = new double[qtd];
            
            // Lê os valores do vetor A
            System.out.println("\n# Digite os valores do vetor A: ");
            for (int i = 0; i < qtd; i++)
            {
                System.out.printf("# Valor [%d]: ", i + 1);
                vetA[i] = ent.nextDouble();
            }
            
            // Lê os valores do vetor B
            System.out.println("\n# Digite os valores do vetor B: ");
            for (int i = 0; i < qtd; i++)
            {
                System.out.printf("# Valor [%d]: ", i + 1);
                vetB[i] = ent.nextDouble();
            }
            
            // Calcula e exibe o vetor resultante C
            System.out.println("\n# VETOR RESULTANTE: ");
            for (int i = 0; i < qtd; i++)
            {
                vetC[i] = vetA[i] + vetB[i];
                System.out.println("-> " + vetC[i]);
            }
        }
    }
}
